pyansys_sphinx_theme/static/css/theme.css

```diff
@@ -179,3 +179,84 @@ p.rubric {
     margin-top: 0.75em;
     margin-bottom: 0.75em;
 }
+
+
+/*
+#############
+*/
+table {
+        width:100%;
+        max-width: 100%; 
+        border-spacing: 0;
+        border-collapse: collapse;
+        overflow: hidden;
+        z-index: 1;
+        font-size: 16px;
+        text-align: center;
+        
+        /* Disabling scroll bars */
+        overflow-y: scroll;
+        scrollbar-width: none; /* Firefox */
+        -ms-overflow-style: none;  /* Internet Explorer 10+ */
+        }
```

Comment:
This file could do with running through a formatter to clean up the whitespace

Reply:
Hi @Andy-Grigg 

I'm not sure how to do that. Can you point me to some resource?

Kind regards,

Reply:
No idea what you mean :-/

Reply:
Oh.... I will no problem!